src/sympc/api.py

```diff
@@ -60,29 +60,7 @@
     ),
 ]
 
-allowed_external_attrs = [
-    ("sympc.store.CryptoStore.get_primitives_from_store", "syft.lib.python.List"),
-    ("sympc.store.CryptoStore.store", "syft.lib.python.Dict"),
-    ("sympc.session.Session.crypto_store", "sympc.store.CryptoStore"),
-    ("sympc.session.Session.init_generators", "syft.lib.python._SyNone"),
-    ("sympc.session.Session.przs_generators", "syft.lib.python.List"),
-    ("sympc.protocol.fss.fss.mask_builder", "sympc.tensor.ShareTensor"),
-    ("sympc.protocol.fss.fss.evaluate", "sympc.tensor.ShareTensor"),
-    ("sympc.protocol.spdz.spdz.mul_parties", "sympc.tensor.ShareTensor"),
-    ("sympc.protocol.spdz.spdz.spdz_mask", "syft.lib.python.Tuple"),
-    ("sympc.protocol.spdz.spdz.div_wraps", "sympc.tensor.ShareTensor"),
-    (
-        "sympc.session.Session.przs_generate_random_share",
-        "sympc.tensor.ShareTensor",
-    ),
-    (
-        "sympc.store.CryptoStore.populate_store",
-        "syft.lib.python._SyNone",
-    ),
-    (
-        "sympc.utils.get_new_generator",
-        "torch.Generator",
-    ),
```

Comment:
Why are these moved?

Reply:
They are added in a separate list.
I split the `allowed_external_attrs` into 3 lists:
- `allowed_external_attrs`
- `share_tensor_attrs`
- `replicated_tensor_attrs`